releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -35,6 +35,7 @@ Azure App Configuration Push extension for Azure DevOps pipeline can be installe
     inputs:
       azureSubscription: '<subscription>'
       AppConfigurationEndpoint: '<store endpoint>'
+      Depth: 1

```

Comment:
This can easily be lost. Seems the sentence above that mentions to use a json object instead should also call out that depth needs to be updated.

Reply:
How about something like "Additionally, the depth property should also be adjusted to match the expected depth of the JSON structure."